gf-guide/requirements.md

```diff
@@ -38,7 +38,8 @@ For example, the hyphen is used to separate the family name from the style name
 - no abbeviation/accronyms
 - no all caps
 - only basic ASCII glyphs (A-Z, a-z, 0-9), ie no hyphens, underscores nor accented letters.
-- union of family name and style name shall no go over 31 characters.
+- the combined lenght of family name and style name shall no go over 32 characters.
```

Comment:
`lenght` typo

Reply:
It's already fixed after the `typo fix 2` commit